src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -480,13 +480,20 @@ private String getUserSessionId(HttpServletRequest request) {
 		try {
 			membersString = restTemplate.getForObject(requestUrl, String.class);
 		} catch (RestClientException e) {
-			errorMessage = "Cannot find site members by siteId: " + site_id
-					+ " " + e.getMessage();
-			log.error(errorMessage);
-			throw e;
+			if (site_id.startsWith(Utils.CTOOLS_SITE_TYPE_MYWORKSPACE_PREFIX))
+			{
+				return getMyWorkspaceSiteMember(sessionId, site_id);
+			}
+			else
+			{
+				errorMessage = "Cannot find site members by siteId: " + site_id
```

Comment:
Log the requestUrl when there is an error unless that is a serious security issue.